test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13154,6 +13150,9 @@ def __init__(self, name=None, attrs={}, pbsconf_file=None, snapmap={},
             self.dflt_config = {'$clienthost': self.server.hostname}
         self.version = None
         self._is_cpuset_mom = None
+
+        # set to True if you like this instance of mom to be reverted to its
+        # default when revert_mom() is called.
```

Comment:
How about 'If this is true, the mom will revert to default.  This is true by default, but can be set to False if required by a test'

Reply:
Sure, that's fine. I've updated the comment and pushed a new commit.